pkg/swap/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM alpine:3.5
+
+# swap command - only minimal Alpine install
+
+# use util-linux to get swapfile utils
+#   at some point, hopefully use our own mkswap and swapon
+RUN apk add --update util-linux
+
+# add the entrypoint and make it executable
+COPY . ./
+RUN chmod +x swap.sh
+RUN ln -s swap.sh swap
+
+# ENTRYPOINT
+ENTRYPOINT swap
```

Comment:
we dont currently support `ENTRYPOINT` in the config file, ie if you specify a `command` it replaces the whole line, so this is probably not much use as you always need to specify arguments. We should probably fix this though...